src/App.js

```diff
@@ -1,29 +1,8 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './Components/Calculator';
 
 function App() {
```

Comment:
- [ ] Please, consider that the App.js component should be class-based, as it is required in this milestone 👍